### DISCUSSION TITLE
Is it possible to create properties with WBI?

### DISCUSSION THREAD

**Discussion by dpriskorn** [LeMyst/WikibaseIntegrator#348] [Q&A] [answered]
...

**Comment by dpriskorn** [comment, marked as answer]
It seems the answer is yes! https://github.com/LeMyst/WikibaseIntegrator/blob/rewrite-wbi/wikibaseintegrator/entities/property.py 🥳

**Comment by LeMyst** [comment]
Yes, like any entity type (Lexeme, Item, mediainfo, Property), everything come from BaseEntity.
